framework/test/jest/unit/specs/controller/application.spec.js

```diff
@@ -60,6 +60,7 @@ describe('Application', () => {
 		it('should validate genesisBlock', () => {
 			// Act
 
+			// eslint-disable-next-line no-new
```

Comment:
If it is completely necessary to use new in this test, I would suggest to add the rule to the Jest test suite as you've done for Mocha.

Reply:
In normal cases, we shouldn't use the constructor to have side effect, that's why it should be removed in general (but this requires too much refactoring)

Reply:
Jest tests are completely new. They should serve as example... Do we plan to refactor them as well?

Reply:
Some of the cases are exceptions, in this case it's using as a check, but 90% of the case shouldn't apply to this.
For exceptional cases, we should disable by line like this, and it's consistent across packages too